main.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+names = []
+
+
+def add_name():
+    print('Press x stop entering a name')
+    print('Enter a name:')
+    while True:
+        name = input()
+
+        if name == 'x':
+            # Check at least two names have been entered
+            if len(names) < 2:
+                print("Please enter at least 2 names")
+                continue
+            break
+        # Check that only letters have been entered
+        elif name.isalpha():
+            names.append(name)
+            print("Enter another name: ")
+            continue
+
+        print("Please only use letters: [a-zA-Z]. Try again.")
```

Comment:
Would it not be cleaner to put the logic for the case where the user enters a non-alphabetic name in the else statement? That way, you could drop the two `continue` statements.